addons/base_import_module/tests/test_module/test.xml

```diff
@@ -34,7 +34,7 @@
         </t>
     </template>
 
-    <template id="contactus_test" name="Contact Form" inherit_id="website.contactus">
```

Comment:
To check but I think you should adapt `addons/base_import_module/tests/test_module/__manifest__.py` to now depend on website_form?

Reply:
Is it mandatory given that this module already depends on website and that website_form is installed automatically with website ?

Reply:
Of course, at least for consistency. And if we ever remove the auto_install of website_form... you should not have to check all module who use website_form to now add that they depend on it... if they depend on it directly, it should be mentioned directly IMO.